internal/node/node_repositories.bzl

```diff
@@ -383,6 +400,7 @@ def node_repositories(package_json, node_version=DEFAULT_NODE_VERSION, yarn_vers
     package_json: a list of labels, which indicate the package.json files that need to be installed.
     node_version: optional; the specific version of NodeJS to install.
     yarn_version: optional; the specific version of Yarn to install.
+    preserve_symlinks: Turn on --node_options=--preserve-symlinks for nodejs_binary and nodejs_test rules
```

Comment:
needs more explanation about how we are switching to this mode as default, failures you might see, how to resolve those, what to do if one of your deps is non-compliant

Reply:
👍